SwifterDemoMac/ViewController.swift

```diff
@@ -28,80 +28,89 @@ import Accounts
 import SwifterMac
 import AuthenticationServices
 
+enum AuthorizationMode {
+    @available(macOS, deprecated: 10.13)
+    case account
+    case browser
+}
+
+let authorizationMode: AuthorizationMode = .browser
+
 class ViewController: NSViewController {
-    let useACAccount = false
-    
+    private var swifter = Swifter(
+        consumerKey: "nLl1mNYc25avPPF4oIzMyQzft",
+        consumerSecret: "Qm3e5JTXDhbbLl44cq6WdK00tSUwa17tWlO8Bf70douE4dcJe2"
+    )
     @objc dynamic var tweets: [Tweet] = []
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        if #available(macOS 10.15, *) {
-            authorizeWithASWebAuthentication()
-        } else if #available(macOS 10.13, *) {
-            authorizeWithWebLogin()
-        } else if useACAccount {
+        switch authorizationMode {
+        case .account:
             authorizeWithACAccountStore()
-        } else {
+        case .browser:
             authorizeWithWebLogin()
         }
     }
 
-    @available(macOS, deprecated: 10.13)
     private func authorizeWithACAccountStore() {
-        let accountStore = ACAccountStore()
-        let accountType = accountStore.accountType(withAccountTypeIdentifier: ACAccountTypeIdentifierTwitter)
-
-        accountStore.requestAccessToAccounts(with: accountType, options: nil) { granted, error in
-            guard granted else {
-                print("There are no Twitter accounts configured. You can add or create a Twitter account in Settings.")
-                return
-            }
-
-            guard let twitterAccount = accountStore.accounts(with: accountType).first as? ACAccount else {
-                print("There are no Twitter accounts configured. You can add or create a Twitter account in Settings.")
-                return
+        if #available(macOS 10.13, *) {
+            self.alert(title: "Deprecated",
+                       message: "ACAccountStore was deprecated on macOS 10.13, please use the OAuth flow instead")
```

Comment:
Put at return statement in the first `if` so you don't have to leave the entire implementation inside the `else` clause.

Reply:
I dealt with this by commit 18ff745.